install/environment.dev.yml

```diff
@@ -1,10 +1,11 @@
 name: zarp
 dependencies:
   - bioconda::bedtools
-  - bioconda::snakemake>=8
   - bioconda::snakefmt
-  - biopython
-  - pyopenssl>=23.2.0
-  - python>=3.11
-  - graphviz
-  - unzip
+  - bioconda::snakemake>=9.8
+  - bioconda::snakemake-executor-plugin-cluster-generic
```

Comment:
Addition of the `snakemake-executor-plugin-cluster-generic` parameter to use in the profiles